Tests/TestingTests/MiscellaneousTests.swift

```diff
@@ -286,6 +286,11 @@ struct MiscellaneousTests {
     #expect(testType.displayName == "Named Sendable test type")
   }
 
+  @Test func `__raw__$raw_identifier_provides_a_display_name`() throws {
+    let test = try #require(Test.current)
+    #expect(test.displayName == "raw_identifier_provides_a_display_name")
```

Comment:
Can we include a test for what `test.name` is as well? Will that string containing the following, or something else?

```
`__raw__$raw_identifier_provides_a_display_name`()
```